storage/boltdb/net-state.go

```diff
@@ -0,0 +1,82 @@
+package boltdb
+
+import (
+	"encoding/json"
+	"errors"
+	"log"
+
+	"github.com/boltdb/bolt"
+)
+
+type File struct {
+	Path  string `json:"path"`
+	Value string `json:"value"`
+}
+
+var (
+	ErrCreatingFileBucket = errors.New("error creating file bucket")
+	ErrFileNotFound       = errors.New("error file not found")
+	ErrIterKeys           = errors.New("error unable to iterate through bucket keys")
+	ErrDeletingFile       = errors.New("error unable to delete file key")
+)
+
+func (client *Client) Put(file File) error {
+	return client.db.Update(func(tx *bolt.Tx) error {
+		b, err := tx.CreateBucketIfNotExists([]byte("files"))
+		if err != nil {
+			return ErrCreatingFileBucket
+		}
+
+		fileKey := []byte(file.Path)
+
+		fileBytes, err := json.Marshal(file)
+		if err != nil {
+			log.Println(err)
+		}
+
+		return b.Put(fileKey, fileBytes)
+	})
+}
+
+func (client *Client) Get(fileKey []byte) (File, error) {
+	var fileInfo File
+	err := client.db.Update(func(tx *bolt.Tx) error {
+		b := tx.Bucket([]byte("files"))
```

Comment:
should this be hardcoded to files ? if so we could probably make this a const at the top of the file